packages/syntax-core/src/Popover/Popover.stories.tsx

```diff
@@ -0,0 +1,310 @@
+import { useState, useRef, useEffect, type ReactElement } from "react";
+import type { StoryObj, Meta } from "@storybook/react";
+import Box from "../Box/Box";
+import Typography from "../Typography/Typography";
+import Button from "../Button/Button";
+import Popover from "./Popover";
+import IconButton from "../../../syntax-core/src/IconButton/IconButton";
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
+import Tooltip from "../Tooltip/Tooltip";
+
+const ContentWithTooltips = () => (
+  <Box width="100%" maxWidth="400px" display="flex" direction="column" gap={2}>
+    <Typography transform="uppercase" color="gray700" size={100}>
+      Weekly Plan{" "}
+      <Tooltip content="Save more by subscribing to our monthly or yearly plans!">
```

Comment:
Spent a lot of time on this last night.  A few commits pushed to address it, latest was https://github.com/Cambly/syntax/pull/303/commits/938df8af43fa20a5324adf07d497bd69fe4f76cb which will show the Arrow component changes that happened in previous commits when that source code got inlined into Tooltip.tsx file.

It's still persisting though:
<img width="411" alt="image" src="https://github.com/Cambly/syntax/assets/1890801/47d9dd4e-ed43-4b8e-bd6f-16a50a680a22">


In this particular case it is because the size of the element is so small that the combined border radius is bigger, so trying to center the button gives this.

I think that this will need to be addressed in a follow-up.  It is an edge case that won't be hit in the product-feature that will use these components.